.github/workflows/image-build-push.yaml

```diff
@@ -9,7 +9,60 @@ on:
     types: [published, edited]
 
 jobs:
-  build-and-push-images:
+  build-and-push-images-linux-arm64:
+    runs-on: 
+      - sh-k8s-runner
```

Comment:
Why do we have two labels here?
Also, let's have arm in the name, so we know the platform